pkg/scale/uint128.go

```diff
@@ -25,7 +25,7 @@ var MaxUint128 = &Uint128{
 func MustNewUint128(in interface{}, order ...binary.ByteOrder) (u *Uint128) {
 	u, err := NewUint128(in, order...)
 	if err != nil {
-		panic(err)
+		panic(fmt.Sprintf("creating Uint128: %s", err))
```

Comment:
nit (feel free to 100% ignore, just a FYI) since this panics, we can easily find the source of the error, so no need to wrap context to panics (a bit like exceptions in other languages... I guess Go sucks in that it's dumb)